nephio-ansible-install/README.md

```diff
@@ -0,0 +1,155 @@
+      edge1: {mgmt_subnet: 172.89.0.0/16, pod_subnet: 10.197.0.0/16, svc_subnet: 10.97.0.0/16}
+      edge2: {mgmt_subnet: 172.90.0.0/16, pod_subnet: 10.198.0.0/16, svc_subnet: 10.98.0.0/16}
+      region1: {mgmt_subnet: 172.91.0.0/16, pod_subnet: 10.199.0.0/16, svc_subnet: 10.99.0.0/16}
+  children:
+    vm:
+      hosts:
+        <ip address of the VM>:
+```
+
+Some customizations are required to tailor the installation to your environment. Edit the inventory/nephio.yaml file where you update:
+
+- cloud_user: the username that is created to access the VM using SSH
+- github_username: your GitHub username
+- github_token: GitHub access token to access GitHub [GitHub personal access token](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+- github_organization: (optional) if you use a GitHub organization for the repositories you should add your GitHub organization here, otherwise it uses the GitHub username
+- gitea_username: your [gitea](https://gitea.io) (local repository) username
+- gitea_password: your [gitea](https://gitea.io) (local repository) password
+
+Note: You can choose between using remote GitHub repositories or local gitea repositories for your Nephio environment by setting either the GitHub variables or the gitea variables.
+
+To start running ansible playbooks an ansible environment is required. Below is an example how to install ansible using a virtual environment. The repository scripts rely on the ansible galaxy community collection
+
+```python
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip
+pip install ansible
+pip install pygithub
+ansible-galaxy collection install community.general
+ansible-galaxy collection install community.docker # required for gitea
+```
+
+## deploy nephio environment
+
+Now that the environment is up an running we can install the Nephio environment
+
+First we create some prerequisites, which installs kubectl, kind, kpt, cni and setup the bash environment
+
+```bash
+ansible-playbook playbooks/install-prereq.yaml
+```
+
+Create the GitHub repository(s) Nephio uses (optional: either choose to run this step for GitHub or steps for gitea below)
+
+```bash
+ansible-playbook playbooks/create-repos.yaml
+```
+
+Create the gitea instance Nephio uses (optional: either choose to run this step for gitea or step for GitHub above)
+
+```bash
+ansible-playbook playbooks/create-gitea.yaml
+```
+
+Create the gitea repository(s) Nephio uses (optional: either choose to run this step for gitea or step for GitHub above)
+
+```bash
+ansible-playbook playbooks/create-gitea-repos.yaml
+```
+
+Next we deploy the kind clusters and install the nephio components
+
+```bash
+ansible-playbook playbooks/deploy-clusters.yaml
+```
+
+Lastly we install the environment manifests we use for the workshop scenario's
+
+```bash
+ansible-playbook playbooks/configure-nephio.yaml
+```
```

Comment:
How about adding commands and expected output that can be executed to verify that installation has succeeded? We could also add screenshots of the nephio webui and/or gitea webui as well